frontend/public/models/index.ts

```diff
@@ -321,6 +321,19 @@ export const ImageStreamTagModel: K8sKind = {
   labelPlural: 'Image Stream Tags'
 };
 
+export const ImageStreamImportsModel: K8sKind = {
```

Comment:
We should verify this isn't showing up in the resource dropdown.

Reply:
It is.  :(  [Blacklist it](https://github.com/openshift/console/blob/master/frontend/public/components/resource-dropdown.tsx#L20)?

Reply:
https://github.com/openshift/console/blob/master/frontend/public/components/resource-dropdown.tsx#L64-L67

I suspect the verbs aren't being filled in for static models. Looks like we have the same problem for ProjectRequest.

Reply:
Jira issue to track: https://jira.coreos.com/browse/CONSOLE-789